front/components/sparkle/AppLayout.tsx

```diff
@@ -188,7 +188,7 @@ export default function AppLayout({
 
             <div
               className={classNames(
-                "flex h-full w-full flex-col",
```

Comment:
Do we have any clue why `height:100%` actually takes more than the full viewport? 🤔